### DISCUSSION TITLE
Running zitadel locally

### DISCUSSION THREAD

**Discussion by you1996** [zitadel/zitadel#1777] [Q&A] [answered]
Hi , want to test zitadel locally for learning purposes, but for me as a beginner the process is not clear,
I'll be grateful if anyone can help me, a quick guide for installation or a link can help me I was stuck now for 3 days.
Thanks all for this great project rich of knowledge.
Thank you .

**Comment by fforootd** [comment, marked as answer]
Yeah we totally get this and we will provide with a guide as soon as we fix https://github.com/caos/zitadel/issues/1768 with https://github.com/caos/zitadel/pull/1754. I think think this will take 2-3 more days.

**Reply by fforootd** [reply]
CC @adlerhurst

**Reply by you1996** [reply]
Thanks for your quick reply @fforootd, very good news thank you

**Reply by adlerhurst** [reply]
Hi @you1996 thanks for reaching out and sorry for the delay. I'm working on a reproducible solution and will mention you in the PR as soon as it's ready.

**Reply by you1996** [reply]
Hi @adlerhurst....thank you 😍

**Reply by adlerhurst** [reply]
@you1996 I wasn't able to mention you in https://github.com/caos/zitadel/pull/1754. 

I hope you can start ZITADEL with [the instructions from the README](https://github.com/caos/zitadel/blob/update-contrib/build/README.md). If there are any questions I'm happy to answer them 🙂

**Reply by you1996** [reply]
Excited to see zitadel running locally, thanks for your great work @adlerhurst !

**Reply by adlerhurst** [reply]
hi @you1996 
We made some changes today on the PR. If you have already run the compose command please prune the `zitadel_`-images before you retry as some images changed.

**Reply by you1996** [reply]
Hi @adlerhurst 
Thanks for this guide, all running as expected, thank you!

**Reply by gc-ss** [reply]
@adlerhurst @fforootd Do you have an updated link to https://github.com/caos/zitadel/blob/update-contrib/build/README.md

It was mentioned by @adlerhurst in https://github.com/caos/zitadel/discussions/1777#discussioncomment-787337

**Reply by adlerhurst** [reply]
Good evening @gc-ss :)
Sorry for the outdated link. The updated link is https://github.com/caos/zitadel/blob/main/guides/quickstart.md

You can find more detailed guides in https://github.com/caos/zitadel/blob/main/guides

**Reply by gc-ss** [reply]
Good evening!

I had a quick look at both the links and it seems none of them needs Kubernetes and can just run with native `docker` and `docker-compose`?

Is that right?

If so I'm unable to get started using the quickstart `docker-compose`

https://github.com/caos/zitadel/issues/2063

**Reply by you1996** [reply]
Hi @gc-ss, i can help you to run zitadel using the quickstart, what is the problem? why you are unable to run it using compose ?

**Reply by gc-ss** [reply]
I really appreciate the response @you1996 

> why you are unable to run it using compose ?

I don't know. I wrote down my notes at https://github.com/caos/zitadel/issues/2063

**Reply by gc-ss** [reply]
I have set up an EC2 node where this can be reproduced. If you give me your public SSH key, I can give you access to it.

I'm online at https://gitter.im/caos/zitadel - we can DM

**Reply by you1996** [reply]
Ok, i'am connected too

**Reply by gc-ss** [reply]
Also, related: https://github.com/caos/zitadel/issues/2064
